src/test/isolation2/sql/copy_interrupt.sql

```diff
@@ -0,0 +1,27 @@
+!\retcode gpconfig -c client_connection_check_interval -v 20s;
+!\retcode gpstop -u;
+
+1: SELECT gp_inject_fault('proc_kill', 'suspend', dbid)
+    FROM gp_segment_configuration 
+    WHERE role = 'p' AND content = -1;
+
+0: CREATE TABLE copy_interrupt_table(a int);
+0&: COPY copy_interrupt_table FROM PROGRAM 'while true; do echo 1; sleep 1; done | cat -';
```

Comment:
how about to add on segment test?
```sql
0: CREATE TABLE copy_interrupt_table(a int) DISTRIBUTED REPLICATED;
0&: COPY copy_interrupt_table FROM PROGRAM 'while true; do echo <SEGID>; sleep 1; done | cat -' ON SEGMENT;
```

Reply:
Should I change the existing test to this, or should I add this as a new test after everything else? (Maybe run it in parallel with the first one?)

Reply:
I think you may add second test for segments